frontend/src/components/_common/share-settings/ShareSettings.jsx

```diff
@@ -58,6 +59,8 @@ export default function ShareSettings({
     setShareState((prev) => ({ ...prev, [name]: checked }));
   };
 
+  const { refetch } = useInfiniteFriendPostList();
```

Comment:
```suggestion
  const { refetch: refetchFriendPostList } = useInfiniteFriendPostList();
```
어떤 데이터를 refetch 해오는지 명확하게 하기 위해 rename해서 사용하는 건 어떤가요?

Reply:
완전 좋아요! 좋은 의견 정말 감사합니다!